client/app/templates/project.hbs

```diff
@@ -37,6 +37,7 @@
     </section>
 
     {{#if @model.landusePackages.length}}
+      <section class="large-padding-top large-margin-bottom ruled-adjacent">
```

Comment:
Thank you for adding these sections. Can you indent the content inside these section tags?
Sorry that you made a good catch and it's causing more work 😛